lib/bolt/transport/ssh/connection.rb

```diff
@@ -99,7 +99,7 @@ def connect
             end
           end
 
-          options[:port] = target.port ? target.port : 22
+          options[:port] = target.port || 22
```

Comment:
Why not set in default ssh config? https://github.com/puppetlabs/bolt/blob/4a50b1500e0a47745c4328d29d3ea06e36d9b731/lib/bolt/config.rb#L45-L47

Reply:
We could, but this follows a similar pattern to what we use in the winrm transport https://github.com/puppetlabs/bolt/blob/master/lib/bolt/transport/winrm/connection.rb#L17-18

Reply:
I think this will override ports set in ssh config.